src/__tests__/app/manage-templates/page.test.tsx

```diff
@@ -0,0 +1,63 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import ManageTemplatesPage from '@app/manage-templates/page';
+import content from '@content/content';
+import { getTemplates } from '@utils/form-actions';
+import { Template } from '@utils/types';
+import { TemplateStatus, TemplateType } from '@utils/enum';
+
+const manageTemplatesContent = content.pages.manageTemplates;
+
+const mockTemplates: Template[] = [
+  {
+    id: '1',
+    version: 1,
+    templateType: TemplateType.NHS_APP,
+    templateStatus: TemplateStatus.NOT_YET_SUBMITTED,
+    name: 'Template 1',
+    message: 'Message',
+    subject: 'Subject Line',
+    createdAt: '2021-01-01T00:00:00.000Z',
+  },
+];
+
+jest.mock('@utils/form-actions');
+
+describe('ManageTemplatesPage', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+  test('renders the page without templates', async () => {
+    render(await ManageTemplatesPage());
```

Comment:
Why as a function and not as a JSX element, like `<ManageTemplatesPage />`?

Reply:
Because it is server rendered (async) and when invoked, it returns a promise rather than a JSX element because it has to make server calls.

Reply:
Oh yeah I forgot about that!